pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1089,27 +1089,142 @@ cdef class Tick(SingleConstructorOffset):
 
 
 cdef class Day(Tick):
+    """
+    Offset ``n`` days.
+
+    Parameters
+    ----------
+    n : int, default 1
+        The number of days represented.
+
+    See Also
+    --------
+    :class:`~pandas.tseries.offsets.DateOffset` : Standard kind of date increment.
+
+    Examples
+    --------
+    You can use the parameter ``n`` to represent a shift of n days.
+    >>> from pandas.tseries.offsets import Day
+    >>> ts = pd.Timestamp(2022, 12, 9, 15)
+    >>> ts.strftime('%a %d %b %Y %H:%M')
```

Comment:
For all the examples, can you not use `strftime`? Just show the result of the repr